### PR TITLE
ci: verify code quality

### PR DESCRIPTION
**Pull request by promops**
Init Dart Code Metrics lints

### CODE REVIEW

**Comment by nxtSwitch** [comment]
Consider adding a standard analyzer and formatter to ensure consistent code formatting and quality.

File: .github/workflows/analyze.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  pull_request:
+    types: [opened, reopened, synchronize]
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
```
[the remark this answers is not in the document]
Comment (promops) [reply .github/workflows/analyze.yaml]: fixed

File: .github/workflows/analyze.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: analyze
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
```
[the remark this answers is not in the document]
Comment (promops) [reply .github/workflows/analyze.yaml]: fixed

**Comment by promops** [comment]
> Consider adding a standard formatter to ensure consistent code formatting.

added

**Review by nxtSwitch — APPROVED** [review]
LGTM!

**Pull request merged**